tests/platform_tests/daemon/test_chassisd.py

```diff
@@ -83,12 +84,13 @@ def collect_data(duthost):
         data = duthost.shell('sonic-db-cli STATE_DB HGETALL "{}"'.format(k))['stdout']
         data = compose_dict_from_cli(data)
         dev_data[k] = data
-    return {'keys': keys, 'data': dev_data}
+    data_dict = {'keys': keys, 'data': dev_data}
+    return OrderedDict(sorted(data_dict.items()))
```

Comment:
This change looks ok .. but wondering is this sorting is the one which really helps. do we really need OrderedDict

Reply:
hi @judyjoseph yes, otherwise the normal dictionary will disorder the sort again.